scooby/report.py

```diff
@@ -93,8 +93,8 @@ def _add_packages(self, packages, optional=False):
 
         # Loop over packages
         for pckg in pckgs:
-            name, version = get_version(pckg, optional)
-            if version is not None:
+            name, version = get_version(pckg)
+            if not (version == 'Could not import' and optional):
```

Comment:
Would it make sense to have the string variables as global variables in `report.py` or class variables for `PythonInfo`? Rather than hardcoding these in a few places?

Reply:
Hm. I am always reluctant to that. But I see the point here. I'll do that!

Reply:
I think class variables (still all caps) make the most sense for this:

```py
class PythonInfo:
    NOT_AVAILABLE_MESSAGE = '...'
    ...
```

Reply:
Ha! Saw it too late. But I like it more, will push a change

Reply:
Nope. `get_version` does not get `PythonInfo`, just the package or the package name...

Reply:
Ah so a global variable would be more appropriate